run.pxi

```diff
@@ -28,57 +28,89 @@
          '~name)))
 
 (defcmd describe
-  "Describe the current project."
+  {:doc "Describe the current project."}
   []
   (p/describe @p/*project*))
 
+(defn show-signature [sigs]
+  (println sigs))
+  ;(loop [_ 0]
+  ;  (when (not (empty? sigs))
+  ;    (print (str "B1"  " " (str/replace (str/replace (str (first sigs) " " ) "[" "") "]" "B2")))
+   ; (recur (rest sigs)))))
 
 (defn showdoc [ns function]
-  (let [name (str (if (not= ns "") (str ns "/") ""))]
+  (let [name (str (if (not= ns "pixie.stdlib") (str ns "/") (str ns "/")))]
   ;loads other possible namespaces in case function is in other namespace
-    (if (= @show-all true) (eval (read-string (str "(require " ns ")"))))
+    (if (= @show-all true) (eval   (read-string (str "(require " ns ")"))))
       
     (let [data (meta (eval (read-string (str name function))))]
+      (print (str "\n  " name function "\n\n\t"))
+      (let [func-name (if (not= (str/index-of function "/") nil) (str function) (str name function))]
+      (print (str/replace 
+                (str/replace
+                  (str/replace
+                    (str/replace  
+                      (str/replace (str/replace (str (:signatures data)) "[" "(") "]" ")")
+                      "))" ")") "((" "(") 
+                      "(" 
+                      (str "(" (str/substring func-name (+ 1 (str/index-of func-name "/"))) " ")) " )" ")")))
```

Comment:
The arguments in the signature are as follows: `[ [ c v] [ c v f] ]`. The string replace are a in order to replace them with brackets and the function name so that the signature becomes: 
`(put! c v)  (put! c v f)`

Lemme see if I can put it more intuitively